src/components/App/AppHead.tsx

```diff
@@ -274,7 +283,7 @@ export const AppHead = (props: IProps) => {
             <LinearProgress id="prog" variant="determinate" value={complete} />
           </div>
         )}
-        {(!busy && !doSave) || complete !== 0 || (
```

Comment:
yes, this construct has messed me up in the past.  I was thinking of it as the times we want this...but it's actually the times we don't want this.